src/main/java/mate/academy/exception/InvalidJwtAuthenticationException.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class InvalidJwtAuthenticationException extends RuntimeException {
+    private final String message;
+    private final HttpStatus httpStatus;
+
+    public InvalidJwtAuthenticationException(String message, HttpStatus httpStatus) {
```

Comment:
use `super(message, cause);` if you want to add your custom field. 
The cause is more important then httpStatus and you do not have it.

Reply:
Ok